### ISSUE TITLE
Error in syntax/listblock.php after updating dokuwiki to Hogfather

### ISSUE THREAD

**Issue by Olodin** [dokufreaks/plugin-creole#21]
Hi,
I use the latest version "date 2019-02-06" and just updated dokuwiki to latest release 2020-07-29 “Hogfather” and the creole plugin does not work anymore.

I get this error in webserver log:

PHP Fatal error:  Class 'Doku_Handler_List' not found in /usr/local/www/dokuwiki/lib/plugins/creole/syntax/listblock.php on line 89, referer: https://beasty.lan.butenas.de/dokuwiki/doku.php/start

Some user report similar error in the dokuwiki forum:
https://forum.dokuwiki.org/d/17852-new-update-has-broken-everything

**Comment by Klap-in** [comment]
DokuWiki is refactored. The class is renamed, and namespaces should be used. e.g. dokuwiki\Parsing\Handler\Lists
The changes are done in https://github.com/splitbrain/dokuwiki/pull/2358/files
For this piece of code no fallback is in place.

**Comment by bmolitor** [comment]
> For this piece of code no fallback is in place.

So what does this mean for non-coder users? Is this something that can be fixed without major rewrites of the plugin (and thus, has a realistic chance to be fixed), or will I need to prepare to translate my wiki pages from creole to dokuwiki syntax if I want to keep getting bugfixes?

**Comment by Klap-in** [comment]
Yes, sorry. This a bit help for the peoples that are known with coding.

**Comment by michitux** [comment]
I think the change should be relatively simple but I won't have time to work on this.

@LarsGit223 You contributed most of the recent changes, do you have time to look at this?
